bundles/org.openhab.binding.onewire/src/test/java/org/openhab/binding/onewire/device/BAE0910Test.java

```diff
@@ -53,7 +54,9 @@ public void setupMocks() {
     @Test
     public void counter() {
         addChannel(CHANNEL_COUNTER, "Number");
-        instantiateDevice();
+        final AbstractOwDevice testDevice = instantiateDevice();
```

Comment:
Consider casting to specific type here instead of multiple times below. The best would be to make `DeviceTestParent` or at least `instantiateDevice` generic, which would help also in other places.

Reply:
Done